applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -299,11 +310,11 @@ double ConstitutiveLawUtilities<TVoigtSize>::CalculateCharacteristicLength(const
 
     for(IndexType i_node = 0; i_node < rGeometry.PointsNumber(); ++i_node)  {
         const array_1d<double, 3>& aux_vector = r_center.Coordinates() - rGeometry[i_node].Coordinates();
-        const double aux_value = inner_prod(aux_vector, aux_vector);
+        double aux_value = inner_prod(aux_vector, aux_vector);
         if(aux_value > radius) radius = aux_value;
     }
 
-    return radius;
+    return std::sqrt(radius);
```

Comment:
This is the definition of radius, but I don't remember why I sued the ^2 of the radius, and there was a reason, because the "original code" has a sqrt is you look in the history

Reply:
:) dont worry